stepactions/fips-operator-check-step-action/0.1/fips-operator-check-step-action.yaml

```diff
@@ -60,6 +60,7 @@ spec:
 
     declare -i count=0
     for related_image in "${related_images[@]}"; do
+      related_image="${related_image//$'\n'/}"
```

Comment:
In testing this locally, the `-t` in the `mapfile -d ' ' -t` on line 41 above did not seem to remove the newline from the final element in the `related_images` array. So I introduced this variable expansion here to strip the newline if it exists.